### DISCUSSION TITLE
about upload media assets

### DISCUSSION THREAD

**Discussion by Xf-asuka-miona-yoda** [requarks/wiki#6563] [Help / Questions]
when i upload the same name assets, it will be updated。
like 1.png(14kb)->1.png(16kb),
i know we can use Asset Folders to avoid it,
but i can not sure everyone know make a folder,how can i achieve my purpose？

**Comment by P4mes4n** [comment]
I don't think it's currently possible to prevent this. There is a PR for this, but it's been open for a while.
https://github.com/requarks/wiki/pull/6017

**Reply by ghost** [reply]
Yes, this issue has been up for a while now. I avoid this by bringing the images directly into the page just by copy paste. I dont use the Import Assets feature anymore, mainly because of this.

**Reply by P4mes4n** [reply]
I'm curious how you achieve this, since none of the editors currently support copy/paste with images? (In VE it sometimes works)

**Comment by ghost** [comment]
I use Visual Editor mostly and I just copy the picture into a word file,
one note file, or into microsoft teams (those are the ones that I found
that work) and then I mark the whole file (dont just click on the picture
itself, that doesnt work) and then copy paste it into the page.

It sounds a bit complicated, but I think its actually even a bit faster
that uploading assets and inserting them.
